js/src/product-feed/review-request/review-request-notice.js

```diff
@@ -35,7 +35,10 @@ const ReviewRequestNotice = ( {
 				'Your account is under cool down period. You can request a new review on %s.',
 				'google-listings-and-ads'
 			),
-			formatDate( glaData.dateFormat, new Date( account.cooldown ) )
+			formatDate(
+				`${ glaData.dateFormat }, ${ glaData.timeFormat }`,
+				new Date( account.cooldown )
+			)
```

Comment:
💅 
I think users hardly ever save the format of date or time in WC Settings as blank, but WC Settings still allows this.

![image](https://user-images.githubusercontent.com/17420811/165492533-d43f2e02-8876-4d8d-a26b-866cbb9fe206.png)

And it could cause the formatting result like:
![image](https://user-images.githubusercontent.com/17420811/165492654-41bcaa6c-f0ee-4c35-9aca-07268f4136c4.png)

Probably this code could share the same logic in
https://github.com/woocommerce/google-listings-and-ads/blob/0ebe02d22d1e9621ef4e96b0aa33e8867b0da364/js/src/product-feed/product-statistics/status-box/sync-status.js#L63-L70

Reply:
To be adjusted in a separated PR 🙏  Thanks.

Reply:
I'm also thinking about if we should allow empty time in GLA. In my mind having blank time is some kinda of patch to don't see maybe the time in the  post date or some kind of dates in orders and products?

So I'm wondering if we should just apply a default format (like h:s) instead of just removing the comma and allowing the user to remove the time that can create confusion to HE in case some customer says the GLA notice says 27 April and he is still unavailable. If I empathise with the HE seems hard for me to think WC settings could be the reason creating the issue.  

What do you think?

Reply:
Usually, I would say it should respect the users' WC Settings. If they set it to blank, it could be considered they want to. But your point is thorough and both the **date** and **time** are important in this use case. So, I think it would be nicer to have fallbacks for the blank formats of WC Settings here.